spec/mandrill_dm/message_spec.rb

```diff
@@ -131,6 +131,16 @@ def new_mail(options = {}, &blk)
       expect(message.global_merge_vars).to eq(global_merge_vars)
     end
 
+    it 'does not use gsub to convert string into JSON' do
```

Comment:
this strings by itself don't cause any problems for JSON.parse unless you use `gsub(/:([a-z_]+)/, '"\\1"').gsub('=>', ':')` to convert string into JSON.

merging?